src/main/java/com/easypost/service/BillingService.java

```diff
@@ -65,20 +65,22 @@ public void fundWallet(String amount, PaymentMethod.Priority priority)
         params.put("amount", amount);
 
         // will attempt to serialize empty JSON to a PaymentMethod.class, that's fine
-        Requestor.request(RequestMethod.POST, String.format("%s/%s/%s/%s", EasyPost.API_BASE,
-                paymentMethodObject.getEndpoint(), paymentMethodObject.getId(), "charges"),
+        Requestor.request(RequestMethod.POST,
+                String.format("%s/%s/%s/%s/%s", client.getApiBase(), client.getApiVersion(),
+                        paymentMethodObject.getEndpoint(), paymentMethodObject.getId(), "charges"),
                 params, PaymentMethod.class, client);
     }
 
     /**
      * List all payment methods for this account.
      *
      * @return an EasyPost.PaymentMethod summary object.
-     * @throws EasyPostException when the request fails or billing has not been set up.
+     * @throws EasyPostException when the request fails or billing has not been set
+     *                           up.
      */
     public PaymentMethod retrievePaymentMethods() throws EasyPostException {
         PaymentMethod response = Requestor.request(RequestMethod.GET,
-                String.format("%s/%s", EasyPost.API_BASE, "payment_methods"), null,
+                String.format("%s/%s/%s", client.getApiBase(), client.getApiVersion(), "payment_methods"), null,
```

Comment:
Since the API base won't change for different API calls (it's not like one API call is going to go to a different base url than every other API call + the fact that API base can't be changed after construction anyway), we can move API base formatting downstream into the `request` function.

API version can change on a per-call basis (some go to Beta, some go to v2, etc), that should remain here as a parameter.

Reply:
Requestor class has it, in line 429. This function is not using the `classURL()` from the `Utilities` file because the endpoint has a different name from the model class name (Billing), that's why we are passing in the `API base` here in the `String.format`

Reply:
Not a fan of the classURL() utility if it's going to cause confusing edge cases like this when we DON'T use it. Won't block, but could do with a re-imagining down the line.